corehq/apps/custom_data_fields/dbaccessors.py

```diff
@@ -1,7 +1,9 @@
 from __future__ import absolute_import
+from __future__ import unicode_literals
+from corehq.util.quickcache import quickcache
 
 
-from __future__ import unicode_literals
+@quickcache(['domain', 'field_type'], timeout=60)
```

Comment:
Whats the argument for caching this? Are custom fields added/updated often enough in a domain to cache this for an hour?

Reply:
This is a 60 second cache, although now that invalidation is more robust, we could probably lengthen it.  This definition itself is updated very rarely, but it's referenced quite frequently, so caching it should make sense.

Reply:
> referenced quite frequently

within the same request? Sorry just trying to understand the intend here. If its cached for 60 seconds only, does not look like its intended for multiple requests.

Reply:
I'm not sure whether it's referenced multiple times within the same request, but that's certainly feasible.  It's used in fixture generation, location upload, and editing or adding a location.  60s was just the original cache timeout here: https://github.com/dimagi/commcare-hq/pull/21172/commits/8c2d3c168055b30bc73f8f622bbad2ce519be203#diff-6f967b14b7d77aca82b0459bab5feac8L123